server/chat-commands.js

```diff
@@ -752,10 +735,7 @@ const commands = {
 
 		return this.sendReply("You have cleared your status message.");
 	},
-	backhelp: [
-		`/back - Marks you as back if you are away.`,
-		`/clearstatus - Clears your status message.`,
-	],
+	backhelp: [`/clearstatus - Clears your status message.`],
```

Comment:
??? Shouldn't this be the `clearstatushelp` and `backhelp` should be the old 'Marks you back if you are away'?